table_foosball/table_foosball_test.rb

```diff
@@ -9,8 +9,59 @@ def setup
     @kenichi = Player.new("Kenichi")
```

Comment:
No test fails if I...

* Remove `@id = SecureRandom.uuid`
* Remove `@timestamp = Time.now`

Other than that, the coverage looks good. I have some concerns about the tight coupling between Games and Players, but we'll get to that a little later once this project grows which will be a good lesson in refactoring and ease of change. Don't dwell on this now! This is a great starting point.